src/js/analytics.js

```diff
@@ -13,14 +13,14 @@ export default (user) => {
 
     const pendoConf = {
         visitor: {
-            id: user.id,
-            internal: user.is_internal,
-            lang: user.locale
+            id: user.identity.internal.account_id,
```

Comment:
Can we just pass in user.identity and user.internal as (data, internal) to simplify?